### PR TITLE
Add contributors

### PR DESCRIPTION
**Pull request by poiresel**
## Description

Adding a quick contribution.md #250 

- [X] Documentation

## Changes

Adding a contributor guide for more information on getting started and using the repo.

## Tests

N/A
## Test Coverage
N/A

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Contributing
+
+Thank you for your interest in contributing! We welcome all contributions no matter their size. Please read along to learn how to get started. More information can be found in each of the README.md as well as  https://docs.airswap.io/ about the individual repos.
```
Comment (alicevhenshaw) [inline CONTRIBUTING.md]: `Please read along...` -> `Please read this guide...`

Comment (alicevhenshaw) [reply CONTRIBUTING.md]: 'More information' - about what?

Comment (alicevhenshaw) [reply CONTRIBUTING.md]: `More information about the individual sub-repos can be found in the relevant README.md, as well as on the [airswap docs site](https://docs.airswap.io/)`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Set up:
+First fork airswap-protocols repository and then clone the project. 
+
+Example:
+
+`git clone https://github.com/airswap/airswap-protocols`
+
+This project uses lerna to handle managing the mono-repo. Lerna is a tool for managing JavaScript projects with multiple packages.
```
Comment (alicevhenshaw) [inline CONTRIBUTING.md]: This project multiple many sub-repos found within `source` and `utils`. We use lerna to handle...

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Run the below command from the root directory to start downloading necessary packages:
+
+`yarn install `
```
Comment (alicevhenshaw) [inline CONTRIBUTING.md]: space after install

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+
+## Running the tests
+A great way to explore the code base is to run the tests.
+
+We can run all tests with:
```
Comment (alicevhenshaw) [inline CONTRIBUTING.md]: We can run all tests from the root with:

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+`yarn test`
+
+
```
Comment (alicevhenshaw) [inline CONTRIBUTING.md]: If you want to run the tests of just 1 sub-repo, then run the same command inside the sub-repo.
e.g. `cd source/swap`, `yarn test`

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+## Code Style
+When multiple people are working on the same body of code, it is important that conforms to a similar style. It often doesn’t matter as much which style, but rather that they conform to one style.
```
Comment (alicevhenshaw) [inline CONTRIBUTING.md]: `it is important that conforms` missing the word 'everyone'

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+To ensure your contribution conforms to the style being used in this project, we require that a linter is run prior to committing to ensure styling.
+
+Run:
+
+```
+yarn lint (linting for the Javascript)
+yarn hint (linting for the Solidity code)
+```
```
Comment (alicevhenshaw) [inline CONTRIBUTING.md]: should be publish our style guide too? To show naming conventions etc?

File: CONTRIBUTING.md (hunk 2 above)
Comment (poiresel) [reply CONTRIBUTING.md]: This project multiple many sub-repos found within source and utils -> This project has multiple sub-repos found within `source` and `utils`.

File: CONTRIBUTING.md (hunk 7 above)
Comment (poiresel) [reply CONTRIBUTING.md]: good call, adding it from notion page into it's own md page,

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
+# Contributing
+
+Thank you for your interest in contributing! We welcome all contributions no matter their size. Please read this guide to learn how to get started. More information about the individual sub-repos can be found in the relevant README.md, as well as on the [airswap docs site](https://docs.airswap.io/).
+
+## Set up:
+First fork airswap-protocols repository and then clone the project. 
+
+Example:
+
+`git clone https://github.com/airswap/airswap-protocols`
```
Comment (alicevhenshaw) [inline CONTRIBUTING.md]: Not sure how helpful this example is as it doesn't fork the repo and instead just clones the original directly?

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+```
+cd source/swap
+yarn test
+```
+
+## Code Style
+When multiple people are working on the same body of code, it is important that everyone conforms to a similar style. It often doesn’t matter as much which style, but rather that they conform to one style.
+
+To ensure your contribution conforms to the style being used in this project, we require that a linter is run prior to committing to ensure styling. We have also documenteded out (Solidity style guide)[SOLIDITY_STYLE_GUIDE.md] in the repo as well.
```
Comment (alicevhenshaw) [inline CONTRIBUTING.md]: typo `documenteded`

File: CONTRIBUTING.md (hunk 8 above)
Comment (poiresel) [reply CONTRIBUTING.md]: git clone https://github.com/<YOUR-GITHUB-USER>/airswap-protocols` as an update

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+```
+cd source/swap
+yarn test
+```
+
+## Code Style
+When multiple people are working on the same body of code, it is important that everyone conforms to a similar style. It often doesn’t matter as much which style, but rather that they conform to one style.
+
+To ensure your contribution conforms to the style being used in this project, we require that a linter is run prior to committing to ensure styling. We have also documented our (Solidity style guide)[SOLIDITY_STYLE_GUIDE.md] in the repo as well.
```
Comment (alicevhenshaw) [inline CONTRIBUTING.md]: Incorrect bracketing types on `(Solidity style guide)[SOLIDITY_STYLE_GUIDE.md]`, that won't be a hyperlink. You need [] around the text and then () around the link

**Pull request merged**